Model/IO/Export/VTU_XML/ParaviewVTKXMLexporter.cpp

```diff
@@ -0,0 +1,147 @@
+#include "ParaviewVTKXMLexporter.h"
+#include <iostream>
+
+
+namespace MeshIS
+{
+	namespace Model
+	{
+
+		using Common::Element_T4;
+		using Common::Element_P6;
+
+		namespace Export
+		{
+
+			void ParaviewVTKXMLExporter::Export(const std::string& absolute_file_path, const CMR& mesh_data)
+			{
+				checkFileToSave(absolute_file_path);
+				fileStream.open(absolute_file_path, std::ios::out | std::ios::trunc);
+				
+				if(!fileStream.good()){
+					std::cout << "Error while opening file.\nEnter will close program." << std::endl;
+					std::cin.get();
+					fileStream.close();
+					exit(0);
```

Comment:
yes i consider but i was having errors while building project so i leave the program in this way.